pkg/translator/loki/logs_to_loki.go

```diff
@@ -63,6 +68,9 @@ func LogsToLoki(ld plog.Logs) (*logproto.PushRequest, *PushReport) {
 				resource := pcommon.NewResource()
 				rls.At(i).Resource().CopyTo(resource)
 
+				// adds level attribute from log.severityNumber
+				addLogLevelAttributeAndHint(log)
```

Comment:
If I'm reading this PR right, this is what the code is doing:

1. if the severity of the record is defined, add a `loki.attribute.labels` hint with "level" as the value
2. adds a new log record attribute with the name "level" 

The outcome is that the translator will then remove both the hint and the record attribute when parsing the level as a label.

I think it would be cleaner to just add the level as part of the merged labels.

Reply:
I think that was the first intention. But was changed after https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14560#discussion_r987768276.

Reply:
Sorry, I missed that comment from @kovrus. It's fine to leave it as he suggested, but please, document the reasoning.